api/jobs_api.py

```diff
@@ -13,15 +13,10 @@ def get_job_all():
 
 @jobs_api.route('/api/jobs', methods=['GET'])
 def get_job():
-    results = jobs_service.get_job(request)
-    return jsonify(results)
-
-
-
-
-
-
-
-
```

Comment:
It is a better idea to get the uuid from the request here and pass it to the service, the service should not care a about the request that much but should get only its uuid and the error handling should
  be done here then